tls/extensions/s2n_npn.c

```diff
@@ -79,3 +79,58 @@ const s2n_extension_type s2n_server_npn_extension = {
     .should_send = s2n_server_npn_should_send,
     .if_missing = s2n_extension_noop_if_missing,
 };
+
+bool s2n_npn_encrypted_should_send(struct s2n_connection *conn)
+{
+    return s2n_server_alpn_should_send(conn);
+}
+
+S2N_RESULT s2n_calculate_padding(uint8_t protocol_len, uint8_t *padding_len)
+{
+    RESULT_ENSURE_REF(padding_len);
+
+    /* https://datatracker.ietf.org/doc/html/draft-agl-tls-nextprotoneg-04#section-3
+     * The length of"padding" SHOULD be 32 - ((len(selected_protocol) + 2) % 32)
+     */
```

Comment:
Are we not doing duvet style compliance comments to avoid adding the draft RFC to our list of duvet RFCs? Does it break duvet somehow?

Reply:
No it doesn't break anything I just don't want to add a draft RFC to our list. lmk if you disagree.